includes/restoreMappings.js

```diff
@@ -40,10 +41,21 @@ class Restore {
         // If the line can't be parsed as JSON it is most likely an incomplete write.
         // If the backup was resumed we can ignore the error because the line will be repeated.
         const parseError = new BackupError('BackupFileJsonError', `Error on line ${backupLine.lineNumber} of backup file - cannot parse as JSON`);
-        // If the backup wasn't resumed then it is invalid and we should error, but we have no way to detect this atm.
-        mappingDebug(`${parseError}`);
-        // Return an empty array if there was an ignorable line
-        return [];
+
+        // Check if the backup was resumed
+        if (backupLine.line.slice(-RESUME_COMMENT.length) === RESUME_COMMENT) {
+          // Try to parse again with the resume marker removed
+          try {
+            arr = JSON.parse(backupLine.line.slice(0, -RESUME_COMMENT.length));
+          } catch (err) {
+            mappingDebug(`${parseError}`);
+            // Return an empty array for an ignorable line
+            return [];
+          }
+        } else {
```

Comment:
```suggestion
        } else if (backupLine.line.charAt(0) !== '\u001E' {
          // The backup file comes from an older version of couchbackup that predates RESUME_COMMENT
          // and json-seq output. For compatibility ignore the broken JSON line assuming it was part of a
          // resume.
          console.error(`Ignoring invalid JSON on line ${backupLine.lineNumber} of backup file as it was written by an older version of couchbackup and could be a valid resume point.`);
          return [];
        } else {
```

Reply:
What about adding a version number range in the error message? It would require opening a new ticket and updating before release once we know the version number
e.g. `Ignoring invalid JSON ... written by couchbackup version < 3.0.0 and could...`

Reply:
If we have more releases before `modernization` merges they'll be patch releases on `2.9` so I think we already know it will be `2.10.0`. I've no objection to adding `< 2.10.0` to the message.

TBH I wasn't 100% sure we should output a message this obviously rather than say a debug line given we are handling it automatically.

Reply:
Fair point, we are changing the behavior to some degree from the current emit an error to print an error to console.  I'm fine with updating the message and update `console.error` -> `mappingDebug`